figures/tasks.py

```diff
@@ -378,6 +390,12 @@ def run_figures_monthly_metrics():
     """
     Populate monthly metrics for all sites.
     """
+    # Waffle Switch
```

Comment:
Minor nit: Please remove this comment line. The following conditional statement should be clear enough